Library/Formula/pmd.rb

```diff
@@ -2,23 +2,27 @@
 
 class Pmd < Formula
   homepage "http://pmd.sourceforge.net/"
-  url "https://downloads.sourceforge.net/project/pmd/pmd/5.1.3/pmd-src-5.1.3.zip"
-  sha1 "da6023d73e078444a7b76e8a7a0cfbeb421221e4"
+  url "https://downloads.sourceforge.net/project/pmd/pmd/5.3.2/pmd-src-5.3.2.zip"
+  sha1 "036cdb7583de3ea0d24e7486bf7b9f9f14bfb1ec"
+
+  depends_on "maven" => :build
+
+  patch :DATA
 
   def install
-    rm Dir["bin/*.{bat,cmd,dll,exe}"]
+    system "mvn", "clean", "package"
 
-    doc.install "LICENSE", "NOTICE", "ReadMe.txt", "licences", Dir["docs/*"]
-    libexec.install "bin", "etc", "lib"
+    doc.install "LICENSE", "NOTICE", "README.md"
 
-    bin.install_symlink "#{libexec}/bin/run.sh" => "pmd"
+    libexec.install "pmd-dist/target/pmd-bin-5.3.2/pmd-bin-5.3.2/bin", "pmd-dist/target/pmd-bin-5.3.2/pmd-bin-5.3.2/lib"
```

Comment:
Let's de-version this so we don't have to change this line every update. `pmd-bin-#{version}` and so on.